src/libs/fsdrv/lv_fs_memfs.c

```diff
@@ -0,0 +1,233 @@
+﻿/**
+ * @file lv_fs_memfs.c
+ *
+ * File System Interface driver for memory-mapped files
+ *
+ * This driver allows using a memory area as a file that can be read by normal file operations. It can
+ * be used, e.g., to store font files in slow flash memory, and load them into RAM on demand.
+ *
+ * You can enable it in lv_conf.h:
+ * 
+ * #define LV_USE_FS_MEMFS 1
+ * 
+ * The actual implementation uses the built-in cache mechanism of the file system interface.
+ *
+ * Since this is not an actual file system, file write and directories are not supported.
+ *
+ * The default drive letter is 'M', but this can be changed in lv_conf.h:
+ *
+ * #define LV_FS_MEMFS_LETTER 'M'
+ *
+ * To use it seamlessly with the file system interface a new extended path object has been introduced:
+ *
+ * lv_fs_path_ex_t mempath;
+ *
+ * This structure can be initialized with the helper function:
+ *
+ * lv_fs_make_path_ex(&mempath, (const uint8_t *) & my_mem_buffer, sizeof(my_mem_buffer));
```

Comment:
Do we really need the `&` before `my_mem_buffer`?

Reply:
Apparently not.